CHANGES.rst

```diff
@@ -63,6 +63,8 @@ Bugfixes
 Internal Changes
 ^^^^^^^^^^^^^^^^
 
+- Add :data:`EMAIL_BACKEND` configuration variable to support different email sending
+  backends e.g. during development (:issue:`5375`, :pr:`5376`, thanks :user:`Moist-Cat`)
```

Comment:
ah i just realized this is at the beginning of the list.. please move it to the end, since we add new changes to the bottom of each section